packages/manager/modules/manager-components/src/region/component.js

```diff
@@ -0,0 +1,11 @@
+import controller from './controller';
+import template from './template.html';
+
+export default {
+  bindings: {
+    regions: '<',
+    model: '=',
```

Comment:
This would oblige all consumers of this component to manually update the model on their side. It could be added later on if it is needed but in this case (netapp) it is not.
I agree that we should pay attention to the bindings use but I'm curious how using a callback is better here, because from first sight it just seems to add complexity in the use (declare a function, update the model, having to pass parameters correctly etc), let me know your thoughts

Reply:
Yes the two-way binding is more simpler to use, but it has its downsides, like adding watchers to the global scope to look for changes, and so may lower performances, even more if it is used many times in the whole application.

Having a callback, indeed bring code complexity, but may limit this effect as it won't be triggered at each scope value update.